SecretSalty/src/main/java/com/secretsalty/secretsalty/repository/UserRepository.java

```diff
@@ -25,75 +25,56 @@ public UserRepository(DSLContext dslContext) {
 
   @Transactional
   public User create(User newUser) {
-    newUser.setCreated(LocalDateTime.now());
-    newUser.setLastModified(LocalDateTime.now());
-
-    AppUserRecord registeredUser =
-        this.dslContext
-            .insertInto(AppUser.APP_USER)
-            .set(this.toRecord(newUser))
-            .returning()
-            .fetchOne();
-
-    return this.toUser(registeredUser);
+    LocalDateTime now = LocalDateTime.now();
```

Comment:
we should maybe put this into a LocalDateTimeUtil class or something like that to share that logic